### ISSUE TITLE
Cannot rebundle

### ISSUE THREAD

**Issue by timdott11** [ValveSoftware/Dota2-Gameplay#11985]
### Cosmetic Item Name

Gambits of Nishai

### Description

I bought set items for Earthshaker (Gambits of Nishai), to rebundle it and use it in tiers upgrade. But i got a message that i cannot rebundle it. Rebundle worked with Enchantress and Clockwerk sets from same treasure tier. I want to know why Gambits of Nishai is not rebundlable and if it is a rule why some bundles and not rebudnlable, except items that not available for trade

### Example Match ID

123456789

### Screenshots

![Снимок экрана 2023-09-14 204519](https://github.com/ValveSoftware/Dota2-Gameplay/assets/145285601/4cbf74da-8494-451d-a293-1da15ef7c463)
![Снимок экрана 2023-09-14 204609](https://github.com/ValveSoftware/Dota2-Gameplay/assets/145285601/c6433c72-6f86-40b2-828f-bbf34aeaf913)

**Comment by slavunya** [comment]
maybe some of item still on trade ban?

**Comment by timdott11** [comment]
> maybe some of item still on trade ban?

no, all items were tradable I waited more than 1 week and was very surprised that I cannot rebundle it, I selled it already. Steam support answered that there are some cosmetic sets that we can't rebundle and I want to know a reason why

**Comment by slavunya** [comment]
> > maybe some of item still on trade ban?
> 
> no, all items were tradable I waited more than 1 week and was very surprised that I cannot rebundle it, I selled it already. Steam support answered that there are some cosmetic sets that we can't rebundle and I want to know a reason why . Message says: "The item is a part of bundle you cannot rebundle"

did you tested with batrider? if works -- do you need to include Loading Screen?

**Comment by timdott11** [comment]
> > > maybe some of item still on trade ban?
> > 
> > 
> > no, all items were tradable I waited more than 1 week and was very surprised that I cannot rebundle it, I selled it already. Steam support answered that there are some cosmetic sets that we can't rebundle and I want to know a reason why . Message says: "The item is a part of bundle you cannot rebundle"
> 
> did you tested with batrider? if works -- do you need to include Loading Screen?

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/145285601/e83c78e4-f592-4305-806a-457319be98b6)

It worked on Enchantress (3 times) and clockwerk (7 times) from same treasure. Earthshaker's set doesn't have a loading screen, I checked many times. It would be another message like "Not all items were found", that I've got when tried to rebundle Dragon Knight's Bitterwing without loading screen, I bought it, waited a week and then rebundled

**Comment by slavunya** [comment]
so loading screens required to rebundle, right?

**Comment by timdott11** [comment]
> so loading screens required to rebundle, right?

Only if bundle has it, I told you about Dragon Knight's Bitterwing bundle, not Earthshaker's Gambits of Nishai, I showed you a screen with bundle page: there is no loading screen
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/145285601/7e4364da-8149-4242-8189-4b95bdf51e96)

**Comment by slavunya** [comment]
> > so loading screens required to rebundle, right?
> 
> Only if bundle has it, I told you about Dragon Knight's Bitterwing bundle, not Earthshaker's Gambits of Nishai, I showed you a screen with bundle page: there is no loading screen ![image](https://user-images.githubusercontent.com/145285601/268922034-7e4364da-8149-4242-8189-4b95bdf51e96.png)

yes I got, thanks, it was additional question, seems Earthshaker's bugged

**Comment by timdott11** [comment]
I hope Techies' bundle wouldn't have this bug :D
It has a loading screen btw :)

**Comment by Natuhaka** [comment]
Now they added a "second style" to the set without even a patch note. It got even buggier because the so-called second style only shows in set preview, not in the item parts.

**Comment by GauravDe-git** [comment]
Any way to bring this set's issue to the devs? I also bought 9 of the set items to trade up not knowing about the bug 😢
